sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -166,22 +166,29 @@ pub fn event_loop<T: OakNode>(
     let mut space = crate::new_handle_space(&read_handles);
 
     loop {
-        // Block until there is a message to read on an input channel.
+        // Block until there is a method notification message to read on an
+        // input channel.
         crate::prep_handle_space(&mut space);
         // TODO: Use higher-level wait function from SDK instead of the ABI one.
         let status =
             unsafe { oak_abi::wait_on_channels(space.as_mut_ptr(), read_handles.len() as u32) };
         crate::result_from_status(status as i32, ())?;
 
         let mut buf = Vec::<u8>::with_capacity(1024);
-        let mut handles = Vec::<Handle>::with_capacity(1);
+        let mut handles = Vec::<Handle>::with_capacity(2);
         crate::channel_read(grpc_in_handle, &mut buf, &mut handles)?;
```

Comment:
Even here, I would use a zero-sized buffer for the initial channel read, and an appropriately-sized buffer for the message reading, I think it would clearer and self-documenting, but up to you

Reply:
It's less efficient but done (if it turns out to be a bottleneck later, we can optimize then).